src/libdmg/con_manager.h

```diff
@@ -0,0 +1,81 @@
+//
+// Created by mattcoding4days on 3/24/21.
+// It is a good idea to separate all DMG's source code into its own folder, essentially
+// acting as a header only library, at least for the TransactionFilter class.
+//
+
+#ifndef BLOCKSEER_LISTENER_BTC_NODE_CON_MANAGER_H
+#define BLOCKSEER_LISTENER_BTC_NODE_CON_MANAGER_H
+
+// standard lib
+#include <string>
+#include <utility>
+#include <memory>
+
+// sql library
+#include <mysql_connection.h>
+#include <mysql_driver.h>
+#include <cppconn/exception.h>
+#include <cppconn/resultset.h>
+#include <cppconn/statement.h>
+
+
+namespace libDMG
+{
+    /* NOTE: _ConnectionDetails should not be used directly
+     * */
+    struct ConnectionDetails
+    {
+    protected:
```

Comment:
Since we don't have a coding standard yet, this is up for debate: My opinion would be that until we have a reason to inherit, we make everything private.

Reply:
Sounds good, this will likely change with the AWS SDK integration. I will make it private and use the object inside ConnectionManager